code_examples/sdk_examples/src/dapp/dapp/domainLinkageCtype.ts

```diff
@@ -1,16 +1,29 @@
 import * as Kilt from '@kiltprotocol/sdk-js'
 
-export const domainLinkageCType: Kilt.ICType = {
-  $id: 'kilt:ctype:0x9d271c790775ee831352291f01c5d04c7979713a5896dcf5e81708184cc5c643',
-  $schema: 'http://kilt-protocol.org/draft-01/ctype#',
-  title: 'Domain Linkage Credential',
-  properties: {
-    id: {
-      type: 'string'
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
hahaha. `eslint` being annoying.

Reply:
yeah and until the fetch function is refactored we can't do much about it